### PR TITLE
Formatting and Refactoring.

### PR DESCRIPTION
**Pull request by amit-webkul**
Use Vue v-text syntax and Laravel colon syntax.

### CODE REVIEW

**Review by suraj-webkul — DISMISSED** [review]

**Review by shivendra-webkul — DISMISSED** [review]

**Review by shivendra-webkul — APPROVED** [review]

**Review by suraj-webkul — DISMISSED** [review]

File: packages/Webkul/Admin/src/Resources/views/catalog/attributes/create.blade.php [hunk 1]
```diff
@@ -274,10 +274,8 @@ class="h-[25px] w-[25px] rounded-md border border-gray-200 dark:border-gray-800"
 
                                                     <!-- Admin-->
                                                     <x-admin::table.td>
-                                                        <p
-                                                            class="dark:text-white"
-                                                            v-text="element.params.admin_name"
-                                                        >
+                                                        <p class="dark:text-white">
+                                                            @{{ element.params.admin_name}}
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/catalog/attributes/create.blade.php:278]: Add space.

File: packages/Webkul/Admin/src/Resources/views/catalog/products/edit/types/downloadable.blade.php [hunk 2]
```diff
@@ -560,7 +634,7 @@ class="grid"
                                                     target="_blank"
                                                     class="break-all text-blue-600 transition-all hover:underline"
                                                 >
-                                                    @{{ element.file_name }}
+                                                    @{{ element.file_name}}
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/catalog/products/edit/types/downloadable.blade.php:637]: Add space.

File: packages/Webkul/Shop/src/Resources/views/products/view/types/bundle.blade.php [hunk 3]
```diff
@@ -33,7 +33,7 @@
                 <ul class="grid gap-2.5 text-base">
                     <li v-for="option in options">
                         <span class="mb-1.5 inline-block">
-                            @{{ option.label }}
+                            @{{ option.label}}
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/products/view/types/bundle.blade.php:36]: Add space,

**Review by devansh-webkul — DISMISSED** [review]

File: packages/Webkul/Admin/src/Resources/views/customers/customers/view/reviews.blade.php [hunk 4]
```diff
@@ -76,6 +76,8 @@ class="align-text-bottom text-base text-gray-800 dark:text-white ltr:ml-1.5 rtl:
                     v-if="available.meta.total"
                     class="grid gap-y-4 border-b p-4 transition-all hover:bg-gray-50 dark:border-gray-800 dark:hover:bg-gray-950"
```
Comment (shivendra-webkul) [inline packages/Webkul/Admin/src/Resources/views/customers/customers/view/reviews.blade.php:77]: Please remove duplicate classes.

File: packages/Webkul/Admin/src/Resources/views/customers/customers/view/reviews.blade.php [hunk 5]
```diff
@@ -74,6 +74,8 @@ class="align-text-bottom text-base text-gray-800 dark:text-white ltr:ml-1.5 rtl:
             <template v-else>
                 <div 
                     v-if="available.meta.total"
+                    class="grid gap-y-4 border-b p-4 transition-all hover:bg-gray-50 dark:border-gray-800 dark:hover:bg-gray-950"
+                    v-for="(record, index) in records"
                     class="grid gap-y-4 border-b p-4 pt-0 transition-all hover:bg-gray-50 dark:border-gray-800 dark:hover:bg-gray-950"
                     v-for="record in available.records"
```
Comment (shivendra-webkul) [inline packages/Webkul/Admin/src/Resources/views/customers/customers/view/reviews.blade.php:80]: Please remove duplicate class attribute and loop.

File: packages/Webkul/Admin/src/Resources/views/components/datagrid/index.blade.php [hunk 6]
```diff
@@ -333,7 +333,9 @@
                     /**
                      * We need to reset the page on filtering.
                      */
-                    this.applied.pagination.page = 1;
+                    if (! appliedColumn.value.length) {
+                        this.applied.filters.columns = this.applied.filters.columns.filter(column => column.index !== columnIndex);
+                    }
```
Comment (shivendra-webkul) [inline packages/Webkul/Admin/src/Resources/views/components/datagrid/index.blade.php:338]: No need to change this. It should be 
this.applied.pagination.page = 1;

Comment (amit-webkul) [reply packages/Webkul/Admin/src/Resources/views/components/datagrid/index.blade.php]: Done

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**